extensions/BMO/bin/export_bmo_etl.pl

```diff
@@ -744,15 +753,21 @@ sub store_cache {
   # We need to use the main DB for write operations
   my $main_dbh = Bugzilla->dbh_main;
 
-  # Clean out outdated JSON
-  $main_dbh->do('DELETE FROM bmo_etl_cache WHERE id = ? AND table_name = ?',
-    undef, $id, $table);
+  try {
+    # Clean out outdated JSON
+    $main_dbh->do('DELETE FROM bmo_etl_cache WHERE id = ? AND table_name = ?',
+      undef, $id, $table);
 
-  # Enter new cached JSON
-  $main_dbh->do(
-    'INSERT INTO bmo_etl_cache (id, table_name, snapshot_date, data) VALUES (?, ?, ?, ?)',
-    undef, $id, $table, $timestamp, $gzipped_data
-  );
+    # Enter new cached JSON
+    $main_dbh->do(
+      'INSERT INTO bmo_etl_cache (id, table_name, snapshot_date, data) VALUES (?, ?, ?, ?)',
+      undef, $id, $table, $timestamp, $gzipped_data
+    );
+  }
```

Comment:
Nit: you could put each of these in their own `try/catch` block, so the error message indicates which part failed for better debugging. The same goes for the other `try/catch` added above.

Reply:
The "$_" in the WARN text will be the actual error message of the command that failed so we will see at which part failed. Since the commands in the try {} block are all dependent on the previous one passing I figured just wrap it all together.